codegen/chllcode.cpp

```diff
@@ -1331,7 +1331,7 @@ void CHLLCode::AddCallStatement(int indLevel, Function *proc, const QString &nam
     QString tgt;
     QTextStream s(&tgt);
     indent(s, indLevel);
-    if (not results->empty()) {
+    if (!results->empty()) {
```

Comment:
Agreed.
On May 19, 2016 7:44 AM, "John Källén" <notifications@github.com> wrote:

> In codegen/chllcode.cpp
> <https://github.com/nemerle/boomerang/pull/62#discussion_r63863656>:
>
> > @@ -1331,7 +1331,7 @@ void CHLLCode::AddCallStatement(int indLevel, Function *proc, const QString &nam
> >      QString tgt;
> >      QTextStream s(&tgt);
> >      indent(s, indLevel);
> > -    if (not results->empty()) {
> > +    if (!results->empty()) {
>
> "If you want Python, you know where to find it" :-)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/nemerle/boomerang/pull/62/files/f972ec39ad93f3ed1c8db753c761cf652f9cdfd3..3616a73e915a265a0bc63ca30a51d32ec561c4e9#r63863656>
>